app/controllers/cases/offender_sar_controller.rb

```diff
@@ -212,15 +212,37 @@ def confirm_sent_to_sscl
       redirect_to case_path(@case) and return
     end
 
-    def confirm_information_received
-      information_received = params[:offender_sar].try(:[], :information_received)
-      return if information_received.present?
+    def accepted_date_received
+      @case = Case::Base.find_by(id: params[:id])
```

Comment:
This should be unnecessary as it's being done in the set_decorated_case before_action. Same with `confirm_accepted_date_received`